app/src/main/java/com/nextcloud/talk/api/NcApi.java

```diff
@@ -727,4 +727,12 @@ Observable<TalkBan> banActor(@Header("Authorization") String authorization,
     @DELETE
     Observable<GenericOverall> unbanActor(@Header("Authorization") String authorization,
                                           @Url String url);
+
+    @POST
+    Observable<GenericOverall> archiveConversation(@Header("Authorization") String authorization,
```

Comment:
would be great to have new api calls implemented in coroutines instead rxjava. 
See `NcApiCoroutines`